src/app/mainview/mainview.component.ts

```diff
@@ -1205,8 +1218,14 @@ export class MainviewComponent implements OnInit, OnDestroy {
         popupWin.document.close();
     }
 
-    /////////////////////// Add/Edit/Delete Scenarios Section ///////////////////////////
+    /////////////////////// Add/Clone/Edit/Delete Scenarios Section ///////////////////////////
```

Comment:
I wonder if we should break these apart for organizational purposes

Reply:
Do you want me to try and break these up?

Reply:
Sure - if it makes sense to do so. It would just would be adding a comment noting each section and then grouping any functions related to it together. That was my thought without looking at how many functions there are, just seemed like a lot to be grouped together.